### PR TITLE
rename charlength and character_length function names

### PR DESCRIPTION
**Pull request by galsalomon66**
Signed-off-by: galsalomon66 <gal.salomon@gmail.com>

### CODE REVIEW

**Comment by galsalomon66** [comment]
some of the tests which uses string-length functions failed since the s3select-engine was updated (to be align with AWS function names)
the like-tests uses much bigger object (which is not that big ..), but in some of teuthology runs it may cause timeout on boto3.
to avoid those random failures in teuthology, the object size was reduced.

**Comment by yuvalif** [comment]
can you please put a link to the PR where the code change is done?

File: s3tests_boto3/functional/test_s3select.py [hunk 1]
```diff
@@ -612,10 +612,6 @@ def test_when_than_else_expressions():
     bucket_name = "test"
     upload_csv_object(bucket_name,csv_obj_name,csv_obj)
 
-    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select  case when ((4*3)==(12)) than "case_1_2" else "case_2_1" end from stdin where (3*3==9);')  ).replace("\n","")
```
Comment (yuvalif) [inline s3tests_boto3/functional/test_s3select.py:615]: why is this test removed?

Comment (galsalomon66) [reply s3tests_boto3/functional/test_s3select.py:615]: the test is wrong , the assert for that test is wrong , it should compare against result of other query.
(in later PR https://github.com/ceph/s3-tests/pull/375 , it is fixed)

**Comment by galsalomon66** [comment]
> can you please put a link to the PR where the code change is done?

https://github.com/ceph/s3select/pull/31/files#diff-df8fff244ae0355c2313b8bf7c2f6e353d791f6ac939620a4e9c80b877519580R88

**Review by yuvalif — APPROVED** [review]

**Pull request merged**